content/projects/gsoc/2019/schedule.adoc

```diff
@@ -0,0 +1,156 @@
+---
+layout: project
+title: "Jenkins year round GSoC schedule"
+tags:
+- gsoc
+- gsoc2019
+---
+
+= Jenkins year round GSoC schedule
+
+== AUG 2018 - JAN 2019
+**Call for GSoC project proposals/mentors**
+
+* period of year where we pro-actively seek for project proposals
+* pre-book SMEs, ask them to make a few hours per coding phase in their schedule (even more so if their SIG participates in GSoC)
+* form mentor groups
+
+**Mentor and project champion recruitment**
+
+* period of the year when we actively search for mentors from various sources like SIGs and mailing list
+
+== SEP 16
+**DevOps Jenkins World San Francisco**
```

Comment:
`DevOps World - Jenkins World` is a correct name. https://wiki.jenkins.io/display/JENKINS/Approved+Trademark+Usage

Reply:
corrected